.github/workflows/no-response.yml

```
@@ -7,6 +7,10 @@ on:
     # Every midnight
     - cron: '0 0 * * *'
 
+permissions:
+  issues: write
+    pull-requests: write
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)